src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -38,6 +38,10 @@ export type KeyboardAwareScrollViewProps = {
   enabled?: boolean;
   /** Adjusting the bottom spacing of KeyboardAwareScrollView. Default is `0` */
   extraKeyboardSpace?: number;
+   /** Used if inverted Flatlist/Flashlist is being used */
+  inverted?: boolean;
+  /** Ajdusting the offset for the content when keyboard is shown on inverted Flatlist/Flashlist on screen with BottomTab navigation */
+  tabBarHeight?: number;
```

Comment:
I think you can re-use `extraKeyboardSpace` property for these purposes? From docs:

<img width="992" alt="image" src="https://github.com/user-attachments/assets/25ba97b1-267a-4f40-a413-952ce1dfcbc7">

Reply:
True